rpmbuild fails on glob bash expansion

`$ docker build --no-cache --file Dockerfile --tag oscap:$(date -u +%Y%m%d%H%M) --tag oscap:latest .`
`$ docker run --cap-drop=all --name scap-security-guide oscap:latest rpm`

```
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Processing files: scap-security-guide-doc-0.1.31-1.el7.centos.noarch
error: File not found by glob: /home/oscap/scap-security-guide/rpmbuild/BUILDROOT/usr/share/doc/scap-security-guide/tables/*.{,x}html
    File not found by glob: /home/oscap/scap-security-guide/rpmbuild/BUILDROOT/usr/share/doc/scap-security-guide/tables/*.{,x}html


RPM build errors:
make: *** [rpm] Error 1
```

```
[root@96d7bf41c869 scap-security-guide]# rpm -qa | grep ^python
python-lxml-3.2.1-4.el7.x86_64
python-chardet-2.2.1-1.el7_1.noarch
python-2.7.5-48.el7.x86_64
python-urlgrabber-3.10-8.el7.noarch
python-iniparse-0.4-9.el7.noarch
python-kitchen-1.1.1-5.el7.noarch
python-libs-2.7.5-48.el7.x86_64
python-pycurl-7.19.0-19.el7.x86_64
```

--- a/scap-security-guide.spec.in
+++ b/scap-security-guide.spec.in
@@ -60,7 +60,8 @@ present in %{name} package.
 
 %files doc
 %doc %{_docdir}/%{name}/guides/*.html
-%doc %{_docdir}/%{name}/tables/*.{,x}html
+%doc %{_docdir}/%{name}/tables/*.html
+%doc %{_docdir}/%{name}/tables/*.xhtml
 
 %changelog
 * __DATE__ __REL_MANAGER__ <__REL_MANAGER_MAIL__> __VERSION__-__RELEASE__

